docs/src/components/ComponentDoc/ComponentExample/ComponentExample.tsx

```diff
@@ -56,7 +56,6 @@ interface ComponentExampleState {
 const childrenStyle: React.CSSProperties = {
   paddingTop: 0,
   paddingBottom: '10px',
-  maxWidth: pxToRem(500),
```

Comment:
Fixes this issue:
![image](https://user-images.githubusercontent.com/14183168/53878850-9cc29700-4015-11e9-9fb1-ea35cd18e3d5.png)
https://stardust-ui.github.io/react/components/animation#types-keyframe-params